src/ioctls/vcpu.rs

```diff
@@ -1157,17 +1157,20 @@ impl VcpuFd {
     /// let vm = kvm.create_vm().unwrap();
     /// let vcpu = vm.create_vcpu(0).unwrap();
     ///
-    /// #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    /// #[cfg(any(target_arch = "x86", target_arch = "x86_64", target_arch = "aarch64"))]
     /// {
     ///     let debug_struct = kvm_guest_debug {
     ///         // Configure the vcpu so that a KVM_DEBUG_EXIT would be generated
     ///         // when encountering a software breakpoint during execution
     ///         control: KVM_GUESTDBG_ENABLE | KVM_GUESTDBG_USE_SW_BP,
     ///         pad: 0,
     ///         // Reset all x86-specific debug registers
+    ///         #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
     ///         arch: kvm_guest_debug_arch {
     ///             debugreg: [0, 0, 0, 0, 0, 0, 0, 0],
     ///         },
+    ///         #[cfg(target_arch = "aarch64")]
+    ///         arch: Default::default(),
```

Comment:
Just curious, can we use the Default impl on both x86 and aarch64 arches?

Reply:
Yep, I just looked the definition of both structure on arm (https://github.com/rust-vmm/kvm-bindings/blob/main/src/arm64/bindings.rs#L1435) and x86 (https://github.com/rust-vmm/kvm-bindings/blob/main/src/x86/bindings.rs#L3011). I agree that you are correct.

Just did the change and updated the PR. PTAL again.

Reply:
Thanks!